Earthfile

```diff
@@ -0,0 +1,64 @@
+
+all:
+    BUILD +all-test
+    BUILD +all-integration-test
+    BUILD +npm
+
+all-test:
+    BUILD --build-arg ELIXIR=1.9.4  --build-arg OTP=20.3.8.26 +test
+    BUILD --build-arg ELIXIR=1.10.4 --build-arg OTP=23.1.1 +test
+    BUILD --build-arg ELIXIR=1.11.0 --build-arg OTP=21.3.8.18 +test
+    BUILD --build-arg ELIXIR=1.11.0 --build-arg OTP=23.1.1 +test
+ 
+test:
+    FROM +setup
+    COPY --dir assets config installer lib integration_test priv test ./
+    RUN mix test
+
+all-integration-test:
+    BUILD --build-arg ELIXIR=1.11.1 --build-arg OTP=21.3.8.18 +integration-test
+    BUILD --build-arg ELIXIR=1.11.1 --build-arg OTP=23.1.1 +integration-test
+
+integration-test:
+    FROM +setup
+    COPY --dir assets config installer lib integration_test priv test ./
+    WORKDIR /src/installer
+    RUN mix deps.get
+
+    WORKDIR /src/integration_test 
+    RUN mix deps.get
+    WITH DOCKER
+        RUN docker run --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 -d postgres && \
+            docker run --name mysql -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -p 3306:3306 -d mysql && \
+            docker run --name mssql -e ACCEPT_EULA=Y -e SA_PASSWORD=some!Password -p 1433:1433 -d mcr.microsoft.com/mssql/server:2019-latest && \
```

Comment:
Could we run the docker compose file instead? Would that be preferred?

Reply:
Yep, a docker compose would work here. It would look like this:
```
WITH DOCKER --compose docker-compose.yml
       RUN  mix test --include database
    END
```
I can make that change if that is your preferred way to structure things.

Reply:
I moved it to docker compose